hack/ginkgo-version.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+gingko_vendor="github.com/onsi/ginkgo/v2"
+if [ -n "$(grep "=> ${gingko_vendor}" go.mod)" ]; then
+	grep "=> ${gingko_vendor}" go.mod | awk '{print $NF}'
+elif [ -n "$(grep "${gingko_vendor}" go.mod)" ]; then
```

Comment:
maybe here no need if, just the cmd

Reply:
I think we need the if else.. if not then both greps might run. I can do an `exit` command but not sure it's any way better..

Reply:
i mean just else

Reply:
ahh ack